cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -101,3 +107,37 @@ func makeModelKeyValue(agentTags map[string]string) []model.KeyValue {
 
 	return tags
 }
+
+func makeSliceOfTags(includedTags string) []string {
+	if includedTags != "" {
+		tags := strings.Split(strings.Trim(includedTags, " "), ",")
```

Comment:
parsing of config parameters should be done close to config / flags code, not in the business type like Reporter